public/Aframe/samples/keypad.js

```diff
@@ -0,0 +1,33 @@
+AFRAME.registerComponent("my_keyboard_controller", {
+    init: function () {
+        this.el.addEventListener("superkeyboardinput", function (event) {
+            if (event.detail.value === "1234") {
```

Comment:
Instead of hardcoding 1234 in here, can you investigate how to have this in a variable that's passed in when initializing? 
Like you know how in the .HTML file we do `super-keyboard="hand: #mouseCursor;"`, similarly, I am imagining something like `my_keyboard_controller="pass: 1234"`, and then in here using that variable. This generalization will help us get the right checks in with the actual number coming from the backend.